### ISSUE TITLE
Model View: Editable dropdown value only updates after hitting enter.

### ISSUE THREAD

**Issue by corivera** [Microsoft/sqlopsstudio#1894]
Let's say I have an editable dropdown with the preset value "OldValue". If I enter "NewValue" in the dropdown, the DropDownComponent.value field will only return "NewValue" after hitting the enter key. I'm trying to use the edit functionality to choose a new value that isn't defined in the dropdown list. Is this behavior by design?

Relying on Enter is problematic, since the wizard framework also uses enter for moving to the next page.

**Comment by MattIrv** [comment]
I think the editable dropdown feature is designed to be more of a "search" than free text input, isn't it?

**Comment by llali** [comment]
yes I think we should implement a new type of drop down which let you add new item and select that. the current one is to search and filter

**Comment by corivera** [comment]
Closing since I can't repro this behavior anymore.

**Comment by llali** [comment]
I verified that Enter makes the value in input box get selected and it doesn't close the dialog.

**Comment by llali** [comment]
we might consider changing the behavior to select the value of the input of lost focus as well
